recipe/patches/0005-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -0,0 +1,142 @@
+From 105366aecb71c918286e99bdf85bf0d04e21bbb5 Mon Sep 17 00:00:00 2001
+From: Kai Fricke <kai@anyscale.com>
+Date: Fri, 3 Dec 2021 11:04:33 -0800
+Subject: [PATCH 5/5] Remove dependencies from setup.py
+
+Signed-off-by: Matti Picus <matti.picus@gmail.com>
```

Comment:
This looks like a left-over (commit message doesn't match generated file-name, and also patch 3 does the same thing).

Reply:
I think you are seeing diff artifacts since I switched patch 3 and patch 5.

Reply:
Perhaps, though despite the reordering (that I'm not against), a commit message of "Remove dependencies from setup.py" whould not lead to a file-name containing "all", if generated according to the patching [mini-guide](https://github.com/conda-forge/ray-packages-feedstock/blob/main/ray-patching.md) using `git format-patch`.

The appearance of _both_ 
```
recipe/patches/0005-Remove-all-dependencies-from-setup.py.patch
recipe/patches/0005-Remove-dependencies-from-setup.py.patch
```
is also suspect

Reply:
whoops, caught me. I edited the patches

Reply:
I redid the patches properly